guests.md

```diff
@@ -13,4 +13,5 @@
 | Susan Sloan       | 3             |
 | Lilian Rasmussen  | 4             |
 | Julian            | 3             |
-| Maxibon           | 5 	          |
+| Maxibon           | 5 	        |
```

Comment:
The last column at the bottom was shifted to the right i fixed it before adding my name, however on git hub it is still showing as shifted.